### ISSUE TITLE
JavaDoc issue with generated build() method

### ISSUE THREAD

**Issue by io7m** [immutables/immutables#207]
Hello.

`build()` methods are generated that have JavaDoc similar to the following:

```
    /**
     * Builds new {@link com.io7m.jcanephora.core.JCGLClearSpecification JCGLClearSpecification}.
     * @return immutable instance of JCGLClearSpecification
     * @throws exception {@code java.lang.IllegalStateException} if any required attributes are missing
     */
    public JCGLClearSpecification build()
        throws IllegalStateException {
      checkRequiredAttributes(); return new JCGLClearSpecification(this);
    }
```

This results in errors because the JavaDoc tool seems to think that the `@throws` tag is referring to a nonexistent type called `exception`. This is the correct form:

```
    /**
     * Builds new {@link com.io7m.jcanephora.core.JCGLClearSpecification JCGLClearSpecification}.
     * @return immutable instance of JCGLClearSpecification
     * @throws IllegalStateException if any required attributes are missing
     */
    public JCGLClearSpecification build()
        throws IllegalStateException {
      checkRequiredAttributes(); return new JCGLClearSpecification(this);
    }
```

Is there some reason for generating the first form, or is this just an oversight?

**Comment by elucash** [comment]
Hi, you're right, we probably have this already fixed in master https://github.com/immutables/immutables/commit/25d17db899f843a845aaebe3462e210325906dd2  I'm planning to make couple of other fixes and then release it. Could you try snapshots or disable strict javadoc in the build in a meanwhile. Thank you!

**Comment by io7m** [comment]
Yep, that looks fixed. Thanks!

**Comment by elucash** [comment]
Fixed by PR #197
